Packs/CaseManagement-Generic/Scripts/CaseMgmtIncidentTypesDisplay/CaseMgmtIncidentTypesDisplay.py

```diff
@@ -0,0 +1,26 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+# Requirements - Create an XSOAR List called IncidentTypesFromList a list of comma separated Incident Types
+# Example: Incident Type 1,Incident Type 2, Incident Type 3
+
+# check if this is a new Incident or not
+incident = demisto.incident().get("id")
+
+# if new Incident, the ID will be empty:
+if not incident:
```

Comment:
Do you know of a case where this condition is met?
Ostensibly, if an incident occurs then it has an id.

Reply:
@israelpoli In this case, if the field-display script executes on a New/Edit form, the ID will be blank.  This is the same logic used by this script:

https://xsoar.pan.dev/docs/reference/scripts/hide-fields-on-new-incident

Also documented here: 
https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-8/cortex-xsoar-admin/incidents/customize-incident-view-layouts/create-dynamic-fields-in-incident-forms